gettsim/data/param.yaml

```
@@ -868,9 +871,36 @@ a2kiz_minek_sin:
     2005-01-01:
       value: 0
       note:
-    2009-01-01:
+    2008-10-01:
       value: 600
-      note:
+      note: G. v. 24.09.2008 BGBl. I S. 1854
+a2kiz_withdrawal_rate:
+  group: kiz
+  name:
+    de: Rate, mit der relevantes Einkommen auf den Kinderzuschlag angerechnet wird. Bis 07/2019 wurde das Einkommen nur in 10€-Schritten berücksichtigt.
+    en: transfer withdrawal of additional child benefit
+  description:
+    de: § 6a (6) BKGG, bis 01.07.2019 § 6a (4) BKGG
+    en:
+  values:
+    2005-01-01:
+      value: 0.7
+      note: Art. 46 v. 24.12.2003, BGBl. I. S. 2954
+    2008-10-01:
+      value: 0.5
+      note: G. v. 24.09.2008 BGBl. I S. 1854
+a2kiz_withdrawal_rate_child:
+  group: kiz
+  name:
+    de: Rate, mit der Einkommen des Kindes auf den Kinderzuschlag angerechnet wird. Parameter ist vor 07/2019 unwirksam, da er in der Funktion nicht auftaucht.
+    en:
+  values:
+    2005-01-01:
+      value: 1
+      note: Art. 46 v. 24.12.2003, BGBl. I. S. 2954
```

```suggestion
      note: Art. 46 G. v. 24.12.2003 BGBl. I. S. 2954.
```
in #69 I decided not to give the exact "Artikel" of the law as it would have been a lot more effort. I think it is nice to add as a service. We could do that consistently everywhere (more effort), not at all (less service for those who double check the law) or just if we want to (less consistency).

Drop the comma behind the date as nothing will show up in the search over at offenegesetze.de if there is a comma (yes it is only a few keystrokes more to remove that, but when you look up a lot of laws it starts to matter).

---

OK with the comma. We need these guidelines on how to update / add params somewhere. Usually, I don't mention the "Art.". The reason is that most interesting changes happen in Art. 1. Here, this was not the case.